### PR TITLE
Remove obsolete classes

### PR DESCRIPTION
**Pull request by darxriggs**
`JenkinsHelper` and `DecoratedLauncher` are not required anymore.

### CODE REVIEW

File: src/main/java/com/cloudbees/jenkins/plugins/customtools/DecoratedLauncher.java [hunk 1]
```diff
@@ -1,95 +0,0 @@
...
-/**
- * A launcher which delegates to a provided inner launcher.
- * Allows subclasses to only implement methods they want to 
- * override.
- * 
- * @author rcampbell
- * @author Oleg Nenashev
- */
-//TODO: 1.567 provides a replacement
-public class DecoratedLauncher extends Launcher {
```
Comment (oleg-nenashev) [inline src/main/java/com/cloudbees/jenkins/plugins/customtools/DecoratedLauncher.java]: This class is still a part of the public API though I do not see any usages in GitHub. I am fine with removing it

**Comment by darxriggs** [comment]
@oleg-nenashev is there anything to do or can it be merged?

**Comment by oleg-nenashev** [comment]
I believe this is okay.
This plugin just needs a maintainer to get it released

**Comment by darxriggs** [comment]
@oleg-nenashev could you at least merge it?
I suggest that you also remove your `<developer>` from the pom.xml.

**Comment by darxriggs** [comment]
@recampbell do you maintain this plugin? At least you are listed as `<developer>` in the pom.xml.

**Comment by oleg-nenashev** [comment]
No, the plugin has no active maintainer.  I can deliver changes on-demand tho

**Comment by oleg-nenashev** [comment]
This one will not hurt anyway

**Comment by darxriggs** [comment]
Thanks.

**Pull request merged**